### PR TITLE
1.37.5

### PR DESCRIPTION
**Pull request by Loloppe**
For new BSML 1.12.0 and BS 1.37.5

### CODE REVIEW

File: Plugin.cs [hunk 1]
```diff
@@ -21,18 +21,17 @@ public void Init(IPALogger logger, Config conf) {
...
-			UI.Manager.Init();
-
-			SharedCoroutineStarter.Init();
-
 			new Harmony("Kinsi55.BeatSaber.BetterSongSearch").PatchAll(Assembly.GetExecutingAssembly());
 		}
 
 		[OnStart]
 		public void OnApplicationStart() {
+			BS_Utils.Utilities.BSEvents.lateMenuSceneLoadedFresh += LateMenuSceneLoadedFresh;
```
Comment (kinsi55) [inline Plugin.cs:29]: Why is this necessary? I'd rather not depend on BS Utils, especially if its just for this

Comment (Loloppe) [reply Plugin.cs:29]: Not really necessary, I'm only using it to get the lateMenuSceneLoadedFresh event, if you got a better way 👍 
Some BSML menu initialization can't be done in Init or OnEnable anymore.

Comment (Loloppe) [reply Plugin.cs]: This is now removed.

**Comment by Loloppe** [comment]
Pressing on download seems to download the song properly, but doesn't display in the list unless restarting the game. Also the play button doesn't work (probably due to the same bug).

**Comment by kinsi55** [comment]
@Loloppe Without having checked anything I'd assume that would be a SongCore regression

**Comment by Loloppe** [comment]
Seems to be all good now, a FieldAccessor had to be changed in SongCore.
